pom.xml

```diff
@@ -220,7 +220,7 @@ THE SOFTWARE.
     <dependency>
       <groupId>org.jenkins-ci.modules</groupId>
       <artifactId>instance-identity</artifactId>
-      <version>3.1</version>
+      <version>201.vd2a_b_5a_468a_a_6</version>
```

Comment:
Is this still relevant, given that we are testing WS agents?

Reply:
I was attempting to align bc dependencies (was bringing `jdk15on` flavor, not `jdk18on`) at the time. I will revert.